### PR TITLE
PUBDEV-5130: Make the default timezone for datetime parsing UTC.

### PR DESCRIPTION
**Pull request by rpeck**

### CODE REVIEW

File: h2o-core/src/test/java/water/parser/ParseTimeTest.java [hunk 1]
```diff
@@ -9,7 +9,10 @@
 import water.fvec.*;
 
 public class ParseTimeTest extends TestUtil {
-  @BeforeClass static public void setup() { stall_till_cloudsize(1); }
+  @BeforeClass static public void setup() {
+    stall_till_cloudsize(1);
+    ParseTime.setTimezone("America/Los_Angeles"); // by default, we now parse in UTC
```
Comment (michalkurka) [inline h2o-core/src/test/java/water/parser/ParseTimeTest.java]: It is better to change the expected results than to force the timezone. Forcing the timezone is a side-effect and can affect other tests.

Comment (rpeck) [reply h2o-core/src/test/java/water/parser/ParseTimeTest.java]: The tests all run independently, no?

Comment (michalkurka) [reply h2o-core/src/test/java/water/parser/ParseTimeTest.java]: They don't. They run on the same cluster that is setup once.

**Comment by rpeck** [comment]
Ok, changed the expected results.

**Pull request merged**